joomla/joomla-cms
# [4.0] Make mediamanager usable on mobile

Pull Request for Issue #27975

### Summary of Changes
Attempting to create working mobile view.


### Testing Instructions
go to administrator/index.php?option=com_media&path=local-0:/
check desktop & mobile view.
apply pr
run `npm run build:com_media`
recheck desktop & mobile views in both portrait & landscape.

Known issues:

- [x] **Currently sidebar doesn't resize as you drill down deeper in desktop view**

- [x] breadcrumb always present.

- [x] breadcrumbs don't wrap properly

- [x] info flyout too narrow in mobile view

- [ ] On some mobile screen file path doesn't wrap.

- [ ] zoom in/out doesn't work properly.  ( sometimes + = out )

### Actual result BEFORE applying this Pull Request
![note8](https://user-images.githubusercontent.com/1850089/96362428-dc5acb80-10f2-11eb-9669-bb0b8840d9e7.jpg)


### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/96644092-2e1b7580-12ee-11eb-847b-e64633bfcdf1.png)
![note8](https://user-images.githubusercontent.com/1850089/96644181-4e4b3480-12ee-11eb-98d0-358e4c5e8bd6.jpg)
![image](https://user-images.githubusercontent.com/1850089/96644216-5905c980-12ee-11eb-884b-c588e61a67cd.png)



### Documentation Changes Required
